test/integration/ember-i18n-csv-test.js

```diff
@@ -56,6 +64,23 @@ describe('integration - ember-i18n-csv', function() {
         });
       });
     });
+    describe('merge', function() {
+      beforeEach(function() {
+        fs.copySync('test/fixtures/locales', 'tmp/locales', { clobber: true });
+      });
+
+      afterEach(function() {
+        fs.emptyDirSync('tmp');
+      });
+
+      it('handles option --merge', function() {
+        return emberI18nCsv('to-js', 'tmp/locales', 'test/fixtures/i18n-merge.csv', { merge: true }).then(() => {
+          return areDirsEqual('tmp/locales', 'test/fixtures/locales').then(areSame => {
```

Comment:
It doesn't look like this is testing anything. The beforeEach is copying 'test/fixtures/locales' into tmp, then you are asserting the result of the merge is equal to what's in 'test/fixtures/locales'. So it appears the merge is having no affect.